invenio_pages/views.py

```diff
@@ -61,11 +61,11 @@ def render_page(path):
     :returns: The rendered template.
     """
     try:
-        page = current_pages_service.read_by_url(g.identity, request.path).to_obj
+        page = current_pages_service.read_by_url(g.identity, request.path).to_dict()
```

Comment:
minor: why would obj not work? maybe we have a serialization problem that needs fixing instead of changing this?

Reply:
to_obj was a hackish property I added to return the record.

Reply:
`to_obj` was returning `self._record`. Doesn't `to_dict()` return other properties as well? In any case, the equivalaent to your `to_obj` property I think it would be [`RecordItem._obj`](https://github.dev/inveniosoftware/invenio-records-resources/blob/a2ab7ab5feedfef480e41821ee01d347883daacf/invenio_records_resources/services/records/results.py#L59) in case you want to play it safe.

Reply:
do we have consensus here ? @ppanero